dev-build/app/Global.scala

```diff
@@ -69,6 +69,7 @@ with ConfigAgentLifecycle
 with SurgingContentAgentLifecycle
 with SectionsLookUpLifecycle
 with ABTHeadlinesLifecycle
+with MostPopularFacebookAutoRefresh.Lifecycle
```

Comment:
The Lifecycle is automatically generated for AutoRefreshes by being a trait inside the base trait. It'll mean we can remove a lot of copy-pasted code.